core/src/main/java/com/sequenceiq/cloudbreak/service/cluster/flow/ClusterTerminationService.java

```diff
@@ -192,8 +192,14 @@ private void deleteFileSystemResources(Long stackId, FileSystem fileSystem) {
                 fsConfiguration.setStorageContainer("cloudbreak" + stackId);
                 fsConfigurator.deleteResources(fsConfiguration);
             }
-        } catch (IOException e) {
-            throw new TerminationFailedException("File system resources could not be deleted: ", e);
+        } catch (IOException ioe) {
```

Comment:
Do we want to handle the IOException as a special case where force delete should be skipped?